src/src/FHSS.h

```diff
@@ -35,42 +35,50 @@ extern int32_t FreqCorrection;
 #define FreqCorrectionMax 200000
 #define FreqCorrectionMin -200000
 
-void ICACHE_RAM_ATTR FHSSsetCurrIndex(uint8_t value);
+
 
 uint8_t ICACHE_RAM_ATTR FHSSgetCurrIndex();
 
+#ifndef SX127x_FREQ_STEP
+#define SX127x_FREQ_STEP 61.03515625
+#endif
+
+#ifndef SX1280_FREQ_STEP
+#define SX1280_FREQ_STEP ((double)(52000000 / pow(2.0, 18.0)))
+#endif
+
 // Our table of FHSS frequencies. Define a regulatory domain to select the correct set for your location and radio
 #ifdef Regulatory_Domain_AU_433
 const uint32_t FHSSfreqs[] = {
-    433420000,
-    433920000,
-    434420000};
+    (uint32_t)((double)433420000/SX127x_FREQ_STEP),
+    (uint32_t)((double)433920000/SX127x_FREQ_STEP),
```

Comment:
Would it make sense to use a macro for this conversion to keep the code a little more readable?